### PR TITLE
Adds FFaker::Address.latitude and longitude

### PR DESCRIPTION
**Pull request by professor**
I'm on a project that has used both Faker and FFaker for a long time. We're now standardizing on FFaker. For much of the code, I'm able to use FFaker API calls. Occasionally, I come across a concept that I'd like to see in FFaker, hence this PR.

This adds a FFaker::Address.latitude and longitude.

### CODE REVIEW

File: lib/ffaker/address.rb [hunk 1]
```diff
@@ -111,5 +111,14 @@ def country_code(given_country = nil)
     def time_zone
       fetch_sample(TIME_ZONE)
     end
+
+    def latitude
+      puts Module.nesting.inspect
```
Comment (Volosh1n) [inline lib/ffaker/address.rb:116]: Is it needed or left after debug purpose?

File: lib/ffaker/address.rb [hunk 2]
```diff
@@ -111,5 +111,14 @@ def country_code(given_country = nil)
...
+      Float((rand * 180) - 90)
```
Comment (Volosh1n) [inline lib/ffaker/address.rb:117]: fetch_sample(-90.0..90.0)
Will it do the trick? Same for the second method

Comment (professor) [reply lib/ffaker/address.rb]: I love it!

File: lib/ffaker/address.rb (hunk 1 above)
Comment (professor) [reply lib/ffaker/address.rb]: Thanks! Too many repos in front of me today.

File: lib/ffaker/address.rb [hunk 3]
```diff
@@ -111,5 +111,13 @@ def country_code(given_country = nil)
     def time_zone
       fetch_sample(TIME_ZONE)
     end
+
+    def latitude
+      Float(fetch_sample(-90, 90))
```
Comment (marocchino) [inline lib/ffaker/address.rb:116]:
```suggestion
      fetch_sample(-90.0..90.0)
```
will be fine.

**Comment by marocchino** [comment]
It would be very helpful if you also update the `Changelog.md`.

File: lib/ffaker/address.rb (hunk 3 above)
Comment (professor) [reply lib/ffaker/address.rb]: Thank you for the suggestion. I'm making this change now. 

I would like latitude and longitude to be like `-130.11409542259364`

**Review by marocchino — APPROVED** [review]
LGTM

**Comment by AlexWayfer** [comment]
Latitude and longitude are more something geographic, location, not address, which is named by aliases.

I've checked do we have such module — and yes, and there are `lat` and `lng` methods: https://github.com/ffaker/ffaker/blob/main/REFERENCE.md#ffakergeolocation

image

**Comment by professor** [comment]
I'm closing this PR. I'd suggest to add an alias for `longitude` and `latitude` to make them easier to find.

**Pull request closed without merging**